components/odh-notebook-controller/run-e2e-test.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+TEST_NAMESPACE="odh-notebook-controller-system"
+
+# setup and deploy the controller
+oc new-project $TEST_NAMESPACE -n $TEST_NAMESPACE --skip-config-write
+IFS=':' read -r -a CTRL_IMG <<< "${ODH_NOTEBOOK_CONTROLLER_IMAGE}"
+export IMG="${CTRL_IMG[0]}"
+export TAG="${CTRL_IMG[1]}"
+make deploy -e K8S_NAMESPACE=$TEST_NAMESPACE
```

Comment:
`make deploy` doesn't work as is in `openshift-ci`

We need to remove dependency on [kubectl](https://github.com/opendatahub-io/kubeflow/blob/v1.6-branch/components/odh-notebook-controller/Makefile#L143) in Makefile. The ci test environment provides `oc`, so we just need to replace the `kubectl` commands with `oc`